erpnext/healthcare/doctype/patient_appointment/patient_appointment.json

```diff
@@ -134,6 +134,7 @@
    "set_only_once": 1
   },
   {
+   "depends_on": "eval:doc.practitioner;",
```

Comment:
Since practitioner is mandatory, is this needed?

Reply:
Yes this is required because you get error if you try to set this field before setting practitioner.